rakelib/lex.rake

```diff
@@ -399,3 +399,44 @@ task "lex:topgems": ["download:topgems", :compile] do
     exit(1)
   end
 end
+
+known_failures = %w[seattlerb/heredoc_nested.txt]
```

Comment:
The tasks themselves are fine, but can you move them into their own rake file? Like `loader.rake` or something.

Reply:
OK